src/pages/rulebook/character_creation/backgrounds.mdx

```diff
@@ -4,30 +4,82 @@ icon: 👨‍🌾
+When creating a character, an optional background can be generated at the same time as you roll for starting coins. Follow this procedure to do so.
 
-This background can be used to inform a Character's Hook and Knacks or can be ignored completely if it doesn't match with what the Player has in mind.
+First, roll a d6 for your character’s social class:
 
-## Socioeconomic Class
+1. Low - The poorest of the poor
+2. Low - Top of the bottom of the barrel
+3. Middle - Your family gets by
+4. Middle - Your family does fine
+5. Upper - Your family is well to do
+6. Noble - The cream of the crop
 
-The first of the three dice rolled for starting coins.
+Multiply the result by 10 to get the first portion of your starting coins.
 
-1. Low
-2. Low
-3. Med
-4. Med
-5. High
-6. Noble
+Next, roll a d6 for your character’s field:
 
-## Background
+1. Outdoor Activities
+2. Culinary Careers
+3. Humble Trades
+4. The Working Class
+5. Respected Professions
+6. Educated Occupations
 
-Second and third dice rolled. First of the two represents the Sphere of backgrounds and the second represents the specific one.
+Again, multiply the result by 10 to get the next portion of your starting coins. Then, roll a final d6 and multiply it by 10 for your final portion of starting coins and then look on the appropriate list below to find your background.
 
-| Sphere                       | 1                 | 2                     | 3                  | 4                    | 5                 | 6                   |
-| :--------------------------- | :---------------- | :-------------------- | :----------------- | :------------------- | :---------------- | :------------------ |
-| **1.** Agrarian & Rural      | Gatherer          | Farmer                | Fisher             | Rancher              | Trapper           | Hunter              |
-| **2.** Culinary Careers      | Miller            | Baker                 | Brewer/Winemaker   | Butcher              | Cook              | Barkeep             |
-| **3.** Humble Trades         | Furrier           | Cobbler/Leatherworker | Fletcher/Ropemaker | Carpenter/Woodcarver | Mason/Stoneworker | Smith/Foundryworker |
-| **4.** The Working Class     | Servant           | Laborer/Teamster      | Peddler/Vendor     | Guard                | Shipmate/Sailor   | Barber/Healer       |
-| **5.** Respected Professions | Tailor            | Apothecary            | Armorer            | Jeweler/Gemcutter    | Shipbuilder       | Architect           |
-| **6.** The Educated Class    | Steward/Navigator | Scribe                | Priest             | Doctor               | Merchant/Captain  | Magistrate/Judge    |
+This background and social class can be used to inform a character's hook and knacks or can be ignored completely if it doesn't match with what the you have in mind.
+
+##Outdoor Activities
```

Comment:
These are not rendering as headings because there's not a space between the last octothorpe and the title.